### PR TITLE
Fixes #8815: Update supported agent for 3.3 - remove 2.11 and 3.0

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8815

### CODE REVIEW

File: release-data/rudder-3.3.cson [hunk 1]
```diff
@@ -74,7 +74,7 @@ components = {
              }
 
 # Agent versions you can use with this version's server
-compatible-agents = [ "2.11", "3.0", "3.1", "3.2" ]
+compatible-agents = [ "3.1", "3.2" ]
 
 # Versions you can upgrade from directly
-supported-upgrade = [ "3.0", "3.1", "3.2" ]
+supported-upgrade = [ "3.1", "3.2" ]
```
Comment (jooooooon) [inline release-data/rudder-3.3.cson]: This should include 3.3 too.

**Comment by amousset** [comment]
Commit modified

File: release-data/rudder-3.3.cson [hunk 2]
```diff
@@ -74,7 +74,7 @@ components = {
              }
 
 # Agent versions you can use with this version's server
-compatible-agents = [ "2.11", "3.0", "3.1", "3.2" ]
+compatible-agents = [ "3.1", "3.2" ]
```
Comment (jooooooon) [inline release-data/rudder-3.3.cson]: And here... :)

Comment (amousset) [inline release-data/rudder-3.3.cson]: Indeed :)

**Comment by amousset** [comment]
Commit modified

**Pull request merged**